package.json

```diff
@@ -1,25 +1,23 @@
 {
   "name": "german-license-plate-codes",
-  "homepage": "https://yuliya-zhukova.github.io/german-license-plate-codes",
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@testing-library/jest-dom": "^5.11.9",
-    "@testing-library/react": "^11.2.5",
+    "@testing-library/jest-dom": "^5.14.1",
+    "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
-    "@types/jest": "^26.0.20",
-    "@types/node": "^12.20.5",
-    "@types/react": "^17.0.3",
-    "@types/react-dom": "^17.0.2",
-    "bootstrap": "^4.6.0",
-    "german-license-plate-prefixes": "^2.17.0",
-    "gh-pages": "^3.1.0",
-    "react": "^17.0.1",
-    "react-bootstrap": "^1.5.2",
-    "react-dom": "^17.0.1",
+    "@types/jest": "^26.0.24",
+    "@types/node": "^12.20.16",
+    "@types/react": "^17.0.14",
+    "@types/react-dom": "^17.0.9",
+    "bootstrap": "^5.0.2",
```

Comment:
Please check manually that the upgrade from Bootstrap 4 to Bootstrap 5 didn't break anything on Windows/Mac/iPhone.